package.json

```diff
@@ -22,7 +22,7 @@
     "bundler"
   ],
   "scripts": {
-    "lint": "eslint \"./bin/*.js\" \"./test/**/*.js\" \"{packages}/**/!(node_modules)/*.js\" ",
+    "lint": "eslint \"./bin/*.js\" \"./packages/**/!(node_modules)/*.js\" ",
```

Comment:
Should we ignore the `test` folder? It's currently in eslintignore.

https://github.com/webpack/webpack-cli/blob/cd890a84facf29b9f9dec7cb876d3b085f4dc9b6/.eslintignore#L3

Reply:
Why we don't want to lint the tests?